resources/assets/js/components/JobDetails/JobDetails.tsx

```diff
@@ -392,6 +392,67 @@ const JobDetails: React.FunctionComponent<
       )
       .required(intl.formatMessage(validationMessages.required)),
   });
+
+  const remoteWorkOptions = [
```

Comment:
I made some changes to this. We had these same magic strings defined in several places, I tried to set up a single source of truth. cb0c08552a4fbcae13377d1a44cf898662098049

Reply:
Looks good to me 👍